i18n/en.i18n.json

```diff
@@ -185,6 +185,8 @@
   "LDAP_Dn" : "Distinguished Name (DN)",
   "LDAP_Enable" : "Enable LDAP",
   "LDAP_Port" : "LDAP Port",
+  "LDAP_Sync_User_Data" : "Keep user data in sync with server",
+  "LDAP_Sync_User_Data_FieldMap" : "User data field map",
```

Comment:
We sure could, unfortunately so many things to tackle, so little daytime :) Feel free to jump in and improve text labels if you please!